cvm-securekey-release-app/AttestationUtil.cpp

```diff
@@ -252,9 +252,7 @@ std::string Util::GetIMDSToken(const std::string &KEKUrl)
     }
 
     curl_easy_cleanup(curl);
-
-    TRACE_OUT("Response: %s\n", Util::reduct_log(responseStr).c_str());
-
+    TRACE_OUT("Response: %s\n", Util::reduct_log(responseStr,Util::reduct_log_percentage(responseStr)).c_str());

```

Comment:
Consider making the call to determine the percentage of log to output in reduct_log() itself.

Reply:
Okay, I do this change.